src/JLSOFile.jl

```diff
@@ -45,8 +46,15 @@ function JLSOFile(
     end
 
     _versioncheck(version, WRITEABLE_VERSIONS)
-    objects = Dict{String, Vector{UInt8}}()
-    jlso = JLSOFile(version, julia, format, compression, image, _pkgs(), objects)
+    jlso = JLSOFile(
+        version,
+        julia,
+        format,
+        compression,
+        image,
+        Pkg.TOML.parse.(_env())...,
```

Comment:
I do not love this but I suspect it is as good as it gets

Reply:
I could save them to variables above if you like?

```
project_toml, manifest_toml = _env()
...
image,
Pkg.TOML.parse(project_toml)
Pkg.TOML.parse(manifest_toml)
...
```